controller/linodemachine_controller.go

```diff
@@ -50,18 +50,18 @@ import (
 )
 
 const (
-	// default etcd disk size in MB
-	defaultEtcdDiskSize      = 10240
+	linodeBusyCode           = 400
+	sizeToMB                 = 1024
```

Comment:
Should this be 1000? For disks, I thought 1GiB = 1024 MiB and 1GB = 1000 MB.

Reply:
good catch, that sounds correct. fixed